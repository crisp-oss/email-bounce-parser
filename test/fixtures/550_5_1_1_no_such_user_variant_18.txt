This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<hazelnut@peoplemadein.co.kr>: host
    kr1-aspmx1.worksmobile.com[152.290.290.215] said: 550 5.1.1 No such user
    ldE8NzE3RyeycidQ-DBxBg - nsmtp (in reply to RCPT TO command)
