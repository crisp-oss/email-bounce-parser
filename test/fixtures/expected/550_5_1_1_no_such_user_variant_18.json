{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<hazelnut@peoplemadein.co.kr>: host kr1-aspmx1.worksmobile.com[152.290.290.215] said: 550 5.1.1 No such user ldE8NzE3RyeycidQ-DBxBg - nsmtp (in reply to RCPT TO command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<hazelnut@peoplemadein.co.kr>: host kr1-aspmx1.worksmobile.com[152.290.290.215] said: 550 5.1.1 No such user ldE8NzE3RyeycidQ-DBxBg - nsmtp (in reply to RCPT TO command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": "5.1.1"
      },
      "label": "No such user ldE8NzE3RyeycidQ-DBxBg - nsmtp",
      "type": "action_not_taken",
      "details": {}
    },
    "recipient": "hazelnut@peoplemadein.co.kr",
    "server": {
      "hostname": "kr1-aspmx1.worksmobile.com",
      "ip": "152.290.290.215",
      "port": null
    },
    "command": "RCPT TO"
  }
}